### PR TITLE
Liam test branch

### PR DESCRIPTION
**Pull request by Crafty15**
Adding stuff for Github demo

### CODE REVIEW

**Review by Bidski — CHANGES_REQUESTED** [review]
PR title does not adequately describe what is being tested

File: Foods.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["pizza", "kale_smoothies", "cheese", "pie"]
```
Comment (Crafty15) [inline Foods.yaml]: Removed Kale smoothies for obvious reasons.

Comment (xXBenjaminYoungXx) [reply Foods.yaml]: Spinach smoothies are the go!

**Review by Crafty15 — COMMENTED** [review]
Changed PR title

File: Foods.yaml [hunk 2]
```diff
@@ -6,4 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["pizza", "kale_smoothies", "cheese", "pie", "Carbonara"]
+  - ["Pizza", "Fermented whale blubber", "Squirrel", "Bratwurst", "kale_smoothies", "cheese", "pie", "Carbonara"]
```
Comment (KipHamiltons) [inline Foods.yaml:9]: it was pointed out to me yesterday that we should try to stick to `snake_case` in `yaml` files in the NUbots codebase
```suggestion
  - ["pizza", "fermented_whale_blubber", "squirrel", "bratwurst", "kale_smoothies", "cheese", "pie", "carbonara"]
```

**Pull request closed without merging**